demos/vue/ckeditor5/README.md

```diff
@@ -0,0 +1,153 @@
+# CKEditor integration in Vue
+
+A simple Vue App integrating WIRIS MathType on a CKEditor 5 and step-by-step information on how to build it. The  code of this example loads a rich text editor instance with a default value.
+
+## Requirements
+
+* npm
+* Vue (*Currently* v11.2.10)
+
+## Recommended IDE Setup
+
+[VSCode](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (and disable Vetur) + [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin).
+
+## How to run the demo
+
+```sh
+$ npm install
+$ npm run dev
```

Comment:
Isn't there a way to use `yarn` + `nx`, as we do in the rest of the demos?

Reply:
Updated :D